src/uct/ib/base/ib_md.c

```diff
@@ -858,6 +864,7 @@ static uct_md_ops_t uct_ib_md_ops = {
     .mem_alloc    = uct_ib_mem_alloc,
     .mem_free     = uct_ib_mem_free,
     .mem_reg      = uct_ib_mem_reg,
+    .mem_detect   = uct_ib_mem_detect,
```

Comment:
why can't use  simply ucs_empty_function_return_success?

Reply:
You are right, I will simplify it now.